packages/server/src/sdk/app/rows/search/external.ts

```diff
@@ -105,37 +106,42 @@ export async function search(
       paginate: paginateObj as PaginationJson,
       includeSqlRelationships: IncludeRelationship.INCLUDE,
     }
-    const queries: Promise<Row[] | number>[] = []
-    queries.push(handleRequest(Operation.READ, tableId, parameters))
+    const queries: [
+      Promise<ExternalReadRequestReturnType>,
+      Promise<number> | undefined
+    ] = [handleRequest(Operation.READ, tableId, parameters), undefined]
     if (countRows) {
-      queries.push(handleRequest(Operation.COUNT, tableId, parameters))
+      queries[1] = handleRequest(Operation.COUNT, tableId, parameters)
```

Comment:
What is this 1? Why not a push?

Reply:
This is just so that I could type the first and second array elements, the second array element is typed as undefined or a count result, so needs replaced - but doesn't matter with Sam's above comment.